Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,63 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string message, params object[] vs)
+        {
+            LogLevel logLevel = LogLevel.Error;
+
+            if (baseLogger == null)
+                throw new ArgumentNullException(baseLogger.ToString());
```

Comment:
This will actually cause a null reference exception when the `baseLogger` is null. The appropriate thing to do is pass the name of the argument that was null.
```suggestion
                throw new ArgumentNullException(nameof(baseLogger));
```